pages/welcome/sections/augur_login_section.py

```diff
@@ -0,0 +1,114 @@
+                            className="instruction_card instruction_card_split",
+                            children=[
+                                html.Div(
+                                    className="instruction_card instruction_card_body",
+                                    children=[
+                                        html.H2("1. Register for an Augur account"),
+                                        html.P(
+                                            """
+                                                After clicking on "Augur log in/sign up" in the top right corner of the page,
+                                                you'll be redirected to the Augur frontend where you can create an account.
+                                            """
+                                        ),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="instruction_card instruction_card_body centered_img",
+                                    children=[
+                                        html.Img(
+                                            className="instruction_image scale_smaller",
+                                            src="assets/welcome_login_section/AugurRegister.png",
+                                        )
+                                    ],
+                                ),
+                            ],
+                        ),
+                        html.Div(
+                            className="instruction_card instruction_card_split",
+                            children=[
+                                html.Div(
+                                    className="instruction_card instruction_card_body",
+                                    children=[
+                                        html.H2("2. Authorize 8Knot to access your user preferences"),
+                                        html.P(
+                                            """
+                                                Once you've created your account, you'll be redirected to an
+                                                authorization page. This page requests that you allow 8Knot
+                                                and Augur to share your account data to enrich your experience.
+                                            """
+                                        ),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="instruction_card instruction_card_body centered_img",
+                                    children=[
+                                        html.Img(
+                                            className="instruction_image",
+                                            src="assets/welcome_login_section/AugurAuthorize.png",
```

Comment:
Image says "http//:0.0.0.0:8050.com"

Reply:
lol yeah it does, didn't notice that.

Reply:
Unsure if I really think this warrants changing. The URL that we share publicly is chaoss.osci.io, but the localhost deployment is pretty generic, and since it's an open source project that somehow seems more appropriate.

Reply:
Makes sense to me